constraintlayout/compose/src/main/java/androidx/constraintlayout/compose/ConstraintLayoutBaseScope.kt

```diff
@@ -384,6 +385,102 @@ abstract class ConstraintLayoutBaseScope {
         return HorizontalAnchor(id, 0, LayoutReferenceImpl(id))
     }
 
+
+    /**
+     * Wrap defines the type of chain
+     */
+    class Wrap private constructor(
```

Comment:
Move these classes to the top level.
Otherwise autocomplete will do `ConstraintLayoutBaseScope.Wrap.Chain`